pkgs/tools/misc/nix-direnv/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, gnugrep, nix }:
+
+stdenv.mkDerivation rec {
+  pname = "nix-direnv";
+  version = "1.0.0";
```

Comment:
The commit message says a different version...?

Reply:
I see the PR title has the fixed version number now (by @Mic92), but the commit message is still `nix-direnv: init at 2020.04.20`.

Reply:
I fixed the commit message. Thank you for checking @bjornfor.